superset-frontend/src/dashboard/actions/dashboardInfo.ts

```diff
@@ -60,10 +64,12 @@ export const setChartConfiguration = (
   });
 
   try {
+    const filterCondition = JSON.parse(String(filterscopes));
```

Comment:
Hey @xinjingqing, I've come across this issue too and found your fix.

Just wanted to note that this line breaks the saving of cross filter scoping since `filterscopes` is `undefined` (called from `CrossFilterScopingModal.tsx`), so it's probably necessary to check this parameter before attempting to `JSON.parse` it.